projects/rocprofiler-systems/source/bin/rocprof-sys-avail/avail.cpp

```diff
@@ -162,6 +162,9 @@ main(int argc, char** argv)
     _category_options.emplace("hw_counters::CPU");
     _category_options.emplace("hw_counters::GPU");
 
+    // Remove unused TIMEMORY tuples
```

Comment:
```suggestion
    // Remove unused TIMEMORY third-party libraries
```
In this context `tpls` is "third-party libraries".